### PR TITLE
add angularjs-nvd3-directives v0.0.7 with git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/angularjs-nvd3-directives/angularjs-nvd3-directives
**Watch 70
  Star 1,077
 Fork 271**
@Amomo could you help me check this PR for #5002? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/angularjs-nvd3-directives/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "angularjs-nvd3-directives",
+  "version": "0.0.7",
+  "filename": "angularjs-nvd3-directives.min.js",
+  "homepage": "http://cmaurer.github.io/angularjs-nvd3-directives",
+  "description": "Angular.js directives for nvd3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cmaurer/angularjs-nvd3-directives.git"
```
Comment (Amomo) [inline ajax/libs/angularjs-nvd3-directives/package.json]: `https://github.com/cmaurer/angularjs-nvd3-directives` will redirect to
`https://github.com/angularjs-nvd3-directives/angularjs-nvd3-directives` now.

Could you make `repository` & `autoupdate` field use the same repo?

**Comment by maruilian11** [comment]
@Amomo i have corrected it, could u help me check it again? thank you!

**Comment by Amomo** [comment]
Good job!
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you~

**Comment by PeterDaveHello** [comment]
We don't need `devDependencies` field in package.json.

**Comment by maruilian11** [comment]
@Amomo i have corrected it, could you help me check it again? thank you!

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you! :-D

**Pull request merged**